e2e/pnpm_workspace_deps/WORKSPACE

```diff
@@ -1,17 +1,8 @@
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
 local_repository(
     name = "aspect_rules_js",
     path = "../..",
 )
 
-http_archive(
-    name = "aspect_bazel_lib",
-    sha256 = "ac6392cbe5e1cc7701bbd81caf94016bae6f248780e12af4485d4a7127b4cb2b",
-    strip_prefix = "bazel-lib-2.6.1",
-    url = "https://github.com/aspect-build/bazel-lib/releases/download/v2.6.1/bazel-lib-v2.6.1.tar.gz",
-)
-
 load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies")
```

Comment:
We probably shouldn't be calling this if we aren't loading it anymore? Can we also remove this? Can we also remove it from MODULE if we can from here?

Reply:
Good catch

Reply:
Some of the MODULE.bazel instances will still be needed if there are load statements to aspect_bazel_lib in that e2e test.. I'll reduce down to the minimal set.